src/styles/card.css

```diff
@@ -0,0 +1,85 @@
+.card,
+.card__product-rate,
+.card__product-price {
+  display: flex;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+
+  justify-content: center;
+  flex-direction: column;
+
+  color: var(--text-primary-color);
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+
+  background-color: var(--card-bg-color);
+  border: 1px solid var(--card-border-color);
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__product-image {
+  align-self: center;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__product-name {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.card__product-code {
+  color: var(--text-secondary-color);
+  margin-bottom: 16px;
+}
+
+.card__product-rate {
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 24px;
+}
+
+.card__product-price {
+  justify-content: space-between;
+  align-items: center;
+  line-height: 18px;
+  margin-bottom: 16px;
+}
+
+.card__product-price-word {
+  color: var(--text-secondary-color);
+}
+
+.card__product-price-amount {
+  font-size: 16px;
+  font-weight: 700;
+}
+
+.card__buy-button {
+  color: #fff;
+  font-size: 14px;
+  font-weight: 700;
+  text-transform: uppercase;
+  text-align: center;
+  text-decoration: none;
+
+  padding: 11px 0;
+  background-color: var(--button-primary-color);
+  border: 1px solid transparent;
+  border-radius: 5px;
+
+  transition: 0.2s ease all;
+}
+
+.card__buy-button:hover {
+  background-color: #fff;
+  border: 1px solid var(--button-primary-color);
```

Comment:
[STYLES] - Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover